lms/djangoapps/bulk_email/tasks.py

```diff
@@ -489,29 +511,67 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
                 sleep(settings.BULK_EMAIL_RETRY_DELAY_BETWEEN_SENDS)
 
             try:
-                log.debug('Email with id %s to be sent to %s', email_id, email)
-
+                log.info(
+                    "BulkEmail ==> Task: %s, SubTask: %s, Email id: %s, Recipient no: %s/%s, \
```

Comment:
nit: If changing stuff, I would suggest changing "recipient no" to just "Recipient:", here and in other messages.